release-notes/opensearch-release-notes-1.0.0.md

```diff
@@ -0,0 +1,517 @@
+# OpenSearch and Dashboards 1.0.0 Release Notes
+
+## Release Highlights
+
+Version 1.0 has multiple enhancements and fixes including span filtering support in Trace Analytics, tenant support in Notebooks, K-NN field level algorithm selection, support for index management transforms, and support for scheduling and tenants in reporting.
+
+## Release Details
+
+OpenSearch and Dashboards 1.0.0 includes the following OpenSearch Migration, features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.0.0.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.0.0.md).
```

Comment:
Do we need to include the `min` here? Seems confusing what these release notes are. @elfisher thoughts?

Reply:
Offline discussion. We are good here

Reply:
We are mentioning the repo here which is not to be confused with products `min`.